### PR TITLE
Differentiate between tested with version and api version

### PR DESCRIPTION
**Pull request by texasaggie97**
- [X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).
- [X] ~~I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.~~
- [X] ~~I've added tests applicable for this pull request~~
### What does this Pull Request accomplish?
* Differentiates between the version that the Python support is tested against and the version of the API Metadata that is used

### What testing has been done?
* Visual inspection of generated files

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -109,7 +109,9 @@ NI-DMM Python API Status
 +-------------------------------+----------------------+
 | NI-DMM (nidmm)                |                      |
 +===============================+======================+
-| Driver Version Tested Against | 19.1.0d11            |
+| Driver Version Tested Against | 18.1.0               |
++-------------------------------+----------------------+
+| Driver API Version            | 19.1.0d11            |
```
Comment (marcoskirsch) [inline README.rst]: This is internal to NI... but ok

File: src/nimodinst/metadata/config_addon.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # We need to maintain the version here since it needs to be updated by the build process on GitHub
 config_additional_config = {
     'module_version': '1.1.1.dev0',
+    'last_tested_version': '18.0.0',
```
Comment (marcoskirsch) [inline src/nimodinst/metadata/config_addon.py]: The name of this key is terrible. It sounds like the last version on Python module `nimodinst` that we tested is 18.0.0.

I think we should take this opportunity to rename this key to something that isn't confusing like `latest_runtime_version_tested_against`

File: README.rst (hunk 1 above)
Comment (texasaggie97) [reply README.rst]: Yes, but the same information shows up in DLL version information.

File: src/nimodinst/metadata/config_addon.py (hunk 2 above)
Comment (texasaggie97) [reply src/nimodinst/metadata/config_addon.py]: Changed key to `latest_runtime_version_tested_against`

**Pull request merged**